### DISCUSSION TITLE
Add an option to autosave/save code in the online editor when working on a kata

### DISCUSSION THREAD

**Discussion by Srbb** [codewars/codewars.com#2644] [Ideas]
I have been working on a challenging kata and progressing a little bit at a time instead of finishing it in one sitting, but since there is no save option whenever my browser closed or the page reloaded without me submitting the code I have lost most of the progress I had made.
Are kata intended to be solved in one sitting? Or should incremental progress and be encouraged?

**Comment by JohanWiltink** [comment]
Your code should be remembered if you `Test`, regardless of passing or failing that test. This even works across multiple devices under the same CW account.

So there already _is_ `Autosave`.
